gcovr/tests/shadow/Makefile

```diff
@@ -0,0 +1,29 @@
+all:
+	cd .build && $(MAKE) -f Makefile all
```

Comment:
I would prefer if the folder `.build` is renamed to e.g. `_build` because in linux in the shell these folders/files starting with dot are hidden (not shown) per default.

Reply:
okay